docs/root_finding.rst

```diff
@@ -68,3 +68,44 @@ Scipy wrapper
   :toctree: _autosummary
 
     jaxopt.ScipyRootFinding
+
+
+Broyden's method
+--------------
+
+.. autosummary::
+  :toctree: _autosummary
+
+    jaxopt.Broyden
+
+Broyden's method is an iterative algorithm suitable for nonlinear root equations in any dimension.
+It is a quasi-Newton method (like L-BFGS), meaning that it uses an approximation of the Jacobian matrix
+at each iteration.
+The approximation is updated at each iteration with a rank-one update.
+This makes the approximation easy to invert using the Sherman-Morrison formula, given it does not use too many
+updates.
+You can control the number of updates with the ``history_size`` argument.
+Furthermore, it uses a line search to ensure the rank-one updates are stable.
+
+Example::
+
+    from jaxopt import Broyden
+
+    def F(x):
+      return x ** 3 - x - 2
+
+    broyden = Broyden(optimality_fun=F, x0=1.0)
```

Comment:
fun=F

Reply:
The initialisation should go in `run`. Please ensure that the example runs correctly :)

Reply:
So sorry about that, they run now